CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Bug Fixes
 - Fix minor bugs for header slicing functionality and expand test coverage for edge-cases. (`#275 <https://github.com/DKISTDC/dkist/pull/275>`_)
 - Fixed inverse transform in `.VaryingCelestialTransformSlit2D`. Which fixes a bug in VISP WCSes. (`#285 <https://github.com/DKISTDC/dkist/pull/285>`_)
 - Fix a bug preventing the transfer of a single dataset with :meth:`~dkist.net.transfer_complete_datasets`. (`#288 <https://github.com/DKISTDC/dkist/pull/288>`_)
+- Implement missing ``select_tag`` method of ``DatasetConverter``. (`#297 <https://github.com/DKISTDC/dkist/pull/297>`_)
```

Comment:
We use towncrier for our changelog, so can you add this as a fragment to the `changelog/` directory? (https://github.com/DKISTDC/dkist/blob/main/changelog/README.rst)

Reply:
Thanks! I added [297.bugfix.rst](c2ed65a0fd9dce3a47a0c4ba0e51ba6c32ec785a) let me know if this should be something other than `bugfix`.